components/brave_wallet_ui/common/hooks/send.ts

```diff
@@ -53,7 +53,8 @@ export default function useSend (isSendTab?: boolean) {
     toAddress,
     toAddressOrUrl,
     showEnsOffchainLookupOptions,
-    ensOffchainLookupOptions
+    ensOffchainLookupOptions,
+    searchingForDomain
```

Comment:
Why aren't we keeping the state in this hook? We're returning `searchingForDomain` from `useSend()` and then using it in `send.tsx`, which defeats the purpose of having the state in Redux.

Reply:
Was following the pattern for this hook implemented in https://github.com/brave/brave-core/pull/12851
This is a good point though, I have moved this to a state directly in the hook.
We should consider refactoring this hook upon removing the now old depreciated send components.